images/debian-python/README.md

```diff
@@ -1,13 +1,17 @@
-#### [`Dockerfile.debian`](Dockerfile.debian)
```

Comment:
Is there any value in adding the SSM agent in case we want to debug inside it?

Reply:
Almost certainly, but outside the scope of this pull request. Maybe we could consider adding this to all of the images we use in CodeBuild?